app/controllers/users.js

```diff
@@ -0,0 +1,31 @@
+const userService = require('../services/users');
+const bcryptService = require('../services/encryption');
+const errors = require('../errors');
+
+const passwordRegex = /^(?=.{8,})(?=.*\d)(?=.*[A-z])(?!.*\s).*$/;
+const emailRegex = /\b[\w\\.-]+@wolox+\.\w{2,4}\b/;
```

Comment:
this regex is able to validate all wolox domain? (wolox.co, wolox.com.ar, wolox.cl)

Reply:
Yes, tested it with all known wolox domains and all of them work